install.ps1

```diff
@@ -877,18 +726,203 @@ if (-not $noGui.IsPresent) {
+    $cancelButton.Font       = New-Object System.Drawing.Font('Microsoft Sans Serif',12)
+    $cancelButton.DialogResult = [System.Windows.Forms.DialogResult]::Cancel
+
+
+    $FormCategories.AcceptButton = $ContinueButton
+    $FormCategories.CancelButton = $cancelButton
+
+    # Read packages to install from the config
+    $packagesToInstall = $configXml.config.packages.package.name
+
+    # Create checkboxes for each package
+    $checkboxesPackages = New-Object System.Collections.Generic.List[System.Object]
+    # Initial vertical position for checkboxes
+    $verticalPosition = 30
+    $numCheckBoxPackages = 1
+    $packages = @()
+    foreach ($category in $packagesByCategory.Keys |Sort-Object) {
+        # Create Labels for categories
+        $labelCategory = New-Object System.Windows.Forms.Label
+        $labelCategory.Text = $category
+        $labelCategory.Font = New-Object System.Drawing.Font('Microsoft Sans Serif',11,[System.Drawing.FontStyle]([System.Drawing.FontStyle]::Bold))
+        $labelCategory.AutoSize = $true
+        $labelCategory.Location = New-Object System.Drawing.Point(10, $verticalPosition)
+        $Panel_Categories.Controls.Add($labelCategory)
+
+        $NumPackages = 0
+        $verticalPosition2 = $verticalPosition + 20
+		$packages= Get-PackagesByCategory -category $category
+		foreach ($package in $packages)
+		{
+			$NumPackages++
+			$checkBox = New-Object System.Windows.Forms.CheckBox
+			$checkBox.Text = $package.PackageName + " : " + $package.PackageDescription
+			$checkBox.Font = New-Object System.Drawing.Font('Microsoft Sans Serif',10)
+			$checkBox.AutoSize = $true
+			$checkBox.Location = New-Object System.Drawing.Point(10, $verticalPosition2)
+			$checkBox.Name = "checkBox$numCheckBoxPackages"
+			$checkboxesPackages.Add($checkBox)
+			$Panel_Categories.Controls.Add($checkBox)
+			$verticalPosition2 += 20
+			$numCheckBoxPackages ++
+		}
+        	# Increment to space checkboxes vertically
+		$verticalPosition += 20 * ($NumPackages ) + 30
+		$numCategories ++
+	}
+
+    # Create empty label and add it to the form categories to add some space
+    $posEnd = $verticalPosition2 +10
+    $emptyLabel                = New-Object system.Windows.Forms.Label
+    $emptyLabel.Width = 20
+    $emptyLabel.Height = 10
+    $emptyLabel.location       = New-Object System.Drawing.Point(10,$posEnd)
+    $Panel_Categories.Controls.Add($emptyLabel)
+
+    # Select packages that are in the config.xml
+    Set-InitialPackages
+
+
+	$FormCategories.controls.AddRange(@($categories_label,$Panel_Categories,$ContinueButton,$resetButton,$allPackagesButton,$cancelButton,$clearPackagesButton))
+	$FormCategories.Add_Shown({$FormCategories.Activate()})
+	$resultCategories = $FormCategories.ShowDialog()
+	if ($resultCategories -eq [System.Windows.Forms.DialogResult]::OK){
+		Install-Selected-Packages
+	} else {
+		Write-Host "[+] Cancel pressed, stopping installation..."
+		Start-Sleep 3
+		exit 1
+	}
```

Comment:
[nitpick] I noticed some tabs used in this file instead of spaces. I'd recommend we pick one and stick with it, though it seems we typically use spaces. We might be able to flag future tab usage with a linter? @Ana06 :eyes: